CMakeLists.txt

```diff
@@ -199,31 +199,30 @@ set(SHARE_DEST share/osc)
 set(GLADE_FILES_DEST ${SHARE_DEST}/glade)
 
 set(PLIB_DEST lib/osc)
-set(SHARE_DEST share/osc)
 
 file(GLOB GLADE_FILES glade/*.glade)
 file(GLOB ICON_FILES icons/*)
 
-install(FILES ${GLADE_FILES} ${ICON_FILES} DESTINATION ${SHARE_DEST})
+install(FILES ${GLADE_FILES} ${ICON_FILES} DESTINATION ${GLADE_FILES_DEST})
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 	configure_file(adi-osc.desktop.cmakein ${CMAKE_CURRENT_BINARY_DIR}/adi-osc.desktop @ONLY)
 	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/adi-osc.desktop
-		DESTINATION /usr/share/applications/)
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications/)
 
 	# install theme icons
 	install(FILES icons/osc16.png RENAME adi-osc.png
-		DESTINATION /usr/share/icons/hicolor/16x16/apps)
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/16x16/apps)
```

Comment:
Ditto... Did you also tested this? I believe icon files are really intended to be in the right places...

Reply:
Ditto.